multipartycall.go

```diff
@@ -25,14 +25,16 @@ type MultiPartyCallAddParticipantParams struct {
 	From                     string `json:"from,omitempty" url:"from,omitempty"`
 	To                       string `json:"to,omitempty" url:"to,omitempty"`
 	CallUuid                 string `json:"call_uuid,omitempty" url:"call_uuid,omitempty"`
+	CallerName               string `json:"caller_name,omitempty" url:"caller_name,omitempty"`
 	CallStatusCallbackUrl    string `json:"call_status_callback_url,omitempty" url:"call_status_callback_url,omitempty"`
 	CallStatusCallbackMethod string `json:"call_status_callback_method,omitempty" url:"call_status_callback_method,omitempty"`
 	SipHeaders               string `json:"sip_headers,omitempty" url:"sip_headers,omitempty"`
 	ConfirmKey               string `json:"confirm_key,omitempty" url:"confirm_key,omitempty"`
 	ConfirmKeySoundUrl       string `json:"confirm_key_sound_url,omitempty" url:"confirm_key_sound_url,omitempty"`
 	ConfirmKeySoundMethod    string `json:"confirm_key_sound_method,omitempty" url:"confirm_key_sound_method,omitempty"`
 	DialMusic                string `json:"dial_music,omitempty" url:"dial_music,omitempty"`
-	RingTimeout              int64  `json:"ring_timeout,omitempty" url:"ring_timeout,omitempty"`
+	RingTimeout              string `json:"ring_timeout,omitempty" url:"ring_timeout,omitempty"`
```

Comment:
type change would break any existing integrations. There are two ways out here

1. Highlight this as breaking change in changelog and README, draft a minor release
2. Or make code accept both String and Integer -> Type should be interface

Same goes for DelayDial